website/blog/2023-01-12-version-071.md

```diff
@@ -0,0 +1,246 @@
+
+## Breaking changes
+
+- **Changes to Console Logging:** `LogBox.ignoreLog` no longer filters console logs. This means you will start seeing logs in the console that you have silenced in LogBox. See [this comment](https://github.com/facebook/react-native/pull/34476#issuecomment-1240667794) for more details.
+- **Removed AsyncStorage and MaskedViewIOS**: These components have been deprecated since version (TODO), so it’s time we remove them entirely. For alternatives, please check [_React Native Directory_](https://reactnative.directory/) for community packages that cover those use cases.
+- **JSCRuntime moved to react-jsc:** react-jsi is now split into react-jsc and react-jsi. If you use JSCRuntime, you will need to add react-jsc as a dependency ([facebook/react-native@6b129d8](https://github.com/facebook/react-native/commit/6b129d81ed8cab301775d2a04971e255df9290de)).
+
+## Acknowledgements
+
+This release is possible thanks to the work of 70+ contributors adding over 1000 commits.
+
+We especially want to thank those who contributed to these major React Native projects:
+
+- **Flexbox Gap Support**: [@intergalacticspacehighway](https://github.com/intergalacticspacehighway) and [@jacobp100](https://github.com/jacobp100).
+- **Web-inspired props**: [@gabrieldonadel](https://github.com/gabrieldonadel) [@dakshbhardwaj](https://github.com/dakshbhardwaj) [@dhruvtailor7](https://github.com/dhruvtailor7) [@ankit-tailor](https://github.com/ankit-tailor) [@madhav23bansal](https://github.com/madhav23bansal).
+- **Codegen Improvements**: [_@AntoineDoubovetzky_](http://github.com/AntoineDoubovetzky), [_@MaeIg_](http://github.com/MaeIg), [_@Marcoo09_](http://github.com/Marcoo09), [_@Naturalclar_](http://github.com/Naturalclar), [_@Pranav-yadav_](http://github.com/Pranav-yadav), [_@ZihanChen-MSFT_](http://github.com/ZihanChen-MSFT), [_@dakshbhardwaj_](http://github.com/dakshbhardwaj), [_@dhruvtailor7_](http://github.com/dhruvtailor7), [_@gabrieldonadel_](http://github.com/gabrieldonadel), [_@harshsiri110_](http://github.com/harshsiri110), [_@ken0nek_](http://github.com/ken0nek), [_@kylemacabasco_](http://github.com/kylemacabasco), [_@matiassalles99_](http://github.com/matiassalles99), [_@mdaj06_](http://github.com/mdaj06), [_@mohitcharkha_](http://github.com/mohitcharkha), [_@tarunrajput_](http://github.com/tarunrajput), [_@vinayharwani13_](http://github.com/vinayharwani13), [_@youedd_](http://github.com/youedd).
+
+## Try out 0.71.0 now!
+
+For React Native CLI users, see the [_upgrade documentation_](https://reactnative.dev/docs/upgrading) for how to update your existing project, or create a new project with `npx react-native init MyProject`.
+
+React Native version 0.71 will be supported in Expo SDK version 48.
+
+_Note: 0.71 is now the latest stable version of React Native and 0.68.x versions are now unsupported. For more information see [React Native’s support policy](https://github.com/reactwg/react-native-releases#releases-support-policy)._
```

Comment:
```suggestion
:::info
0.71 release is now the latest stable version of React Native and 0.68.x versions are now unsupported. For more information see [React Native’s support policy](https://github.com/reactwg/react-native-releases#releases-support-policy)._
:::
```

We should always use annotation before blockquotes for marking content parts. Blockquote was an alternative when we do not have the admonitions, and in a long term will lose the styling and become the quotes they should be.